### ISSUE TITLE
Add X-wing Plug-in to the content

### ISSUE THREAD

**Issue by Hinny** [Hinny/strange-eons-xwing-plug-in#49]
http://basement.cgjennings.ca/Content+Guide

**Comment by kalhamaar** [comment]
Apparently it request a "Passcode to register" provided by strange eon's system admin, do you have one ?

**Comment by Hinny** [comment]
No, I don't think so... I have a profile on the forums and that's it.
Chris Jennings is the creator of Strange Eons and is probably the system admin as well.
